### PR TITLE
Kongmap

### PR DESCRIPTION
**Pull request by nikirago**
Created new KongMap integration to be added to Konghub

### CODE REVIEW

File: app/_hub/yesinteractive/kongmap/index.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Visually Map and Manage your Kong Clusters
+
+description: |
+  Kongmap is a free visualization tool which allows you to view and edit configurations of your Kong API Gateway Clusters, including Routes, Services, and Plugins/Policies. The tool is being offered for installation via Docker and Kubernetes at this time.
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:13]: https://www.grammarly.com/blog/which-vs-that/#:~:text=In%20a%20defining%20clause%2C%20use,and%20you%20can%20use%20which.

```suggestion
  Kongmap is a free visualization tool that allows you to view and edit configurations of your Kong API Gateway Clusters, including Routes, Services, and Plugins/Policies. The tool is being offered for installation using Docker and Kubernetes at this time. 
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+#### Cluster View
+Allows an admin to view a dynamic map of their Kong API Gateway clusters and visually see relationships between
+Workspaces (for Kong Enterprise), Services, Routes (Endpoints), and Plugins (Policies). Clicking on any entity displays
+details of the entity and related links. Plugins can be toggled from view. 
+
+
+![alt text](https://github.com/yesinteractive/kong-map/blob/main/screenshots/kongmap-home.png?raw=true "kongmap")
+
+#### Endpoint Analyzer
+View details of an API Endpoint (Route). The analyzer shows the Service attached to the endpoint/route as well as provides
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:60]:
```suggestion
View details of an API Endpoint (Route). The analyzer shows the Service attached to the endpoint/route as well as 
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+a breakdown of all plugins/policies in order of execution attached to the route/endpoint. For Kong Enterprise users,
+all entities can be viewed directly via a link to Kong Manager.
+
+![alt text](https://github.com/yesinteractive/kong-map/blob/main/screenshots/kongmap-endpoint.png?raw=true "kongmap")
+
+
+#### Declarative Configuration Viewer/Editor
+KongMap is deployed with a browser based version of Kong's CLI tool, decK. Here you can view, edit, and export Kong declarative configurations for your open source
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:68]:
```suggestion
KongMap is deployed with a browser-based version of Kong's CLI tool, decK. Here you can view, edit, and export Kong declarative configurations for your open source 
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+a breakdown of all plugins/policies in order of execution attached to the route/endpoint. For Kong Enterprise users,
+all entities can be viewed directly via a link to Kong Manager.
+
+![alt text](https://github.com/yesinteractive/kong-map/blob/main/screenshots/kongmap-endpoint.png?raw=true "kongmap")
+
+
+#### Declarative Configuration Viewer/Editor
+KongMap is deployed with a browser based version of Kong's CLI tool, decK. Here you can view, edit, and export Kong declarative configurations for your open source 
+and Enterprise clusters via YAML. Declarative
+configuration editing can be disabled by KongMap configuration, or managed via RBAC permissions if using Kong Enterprise.
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:70]:
```suggestion
configuration editing can be disabled by KongMap configuration, or managed with RBAC permissions if using Kong Enterprise. 
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+#### Declarative Configuration Viewer/Editor
+KongMap is deployed with a browser based version of Kong's CLI tool, decK. Here you can view, edit, and export Kong declarative configurations for your open source 
+and Enterprise clusters via YAML. Declarative
+configuration editing can be disabled by KongMap configuration, or managed via RBAC permissions if using Kong Enterprise. 
+
+![alt text](https://github.com/yesinteractive/kong-map/blob/main/screenshots/kongmap-deck.png?raw=true "kongmap")
+
+
+Full documentation available online here: [https://github.com/yesinteractive/kong-map/](https://github.com/yesinteractive/kong-map/)
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:75]:
```suggestion
Full documentation is available here: [https://github.com/yesinteractive/kong-map/](https://github.com/yesinteractive/kong-map/)
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+## Compatibility ## 
+KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.5.
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:78]:
```suggestion
KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.5.x.
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+## Docker Installation ##
+
+Docker image is Alpine 3.11 based running PHP 7.3 on Apache. The container exposes both ports 80 an 443 with a self signed certificated.
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:82]:
```suggestion
Docker image is Alpine 3.11 based running PHP 7.3 on Apache. The container exposes both ports 80 an 443 with a self-signed certificated. 
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+## Compatibility ## 
+KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.5.
+
+## Docker Installation ##
+
+Docker image is Alpine 3.11 based running PHP 7.3 on Apache. The container exposes both ports 80 an 443 with a self signed certificated. 
+
+#### 1. Export Cluster Configurations to Variable ####
+
+The connections to your Kong clusters are defined via JSON. The below example illustrates adding two Kong clusters to KongMap:
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:86]: Most style guides frown upon "via", which is Latin.

```suggestion
The connections to your Kong clusters are defined usin JSON. The below example illustrates adding two Kong clusters to KongMap:
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+#### 3. Authentication ####
+
+If you want to enable authentication to KongMap's UI, it is recommended to run Kongmap behind your Kong Gateway and implement any authentication
+policies you feel is appropriate (OIDC, OAUTH2, Basic Auth, etc.) at the gateway.
+
+## Feedback and Issues
+
+If you have questions, feedback or want to submit issues, please do so here: [https://github.com/yesinteractive/kong-map/issues](https://github.com/yesinteractive/kong-map/issues).
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:138]:
```suggestion
If you have questions, feedback, or want to submit issues, please do so here: [https://github.com/yesinteractive/kong-map/issues](https://github.com/yesinteractive/kong-map/issues).
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+  -e "KONGMAP_URL=http://url_to_kongmap" \
+  -p 8100:80 \
+  -p 8143:443 \
+  yesinteractive/kongmap
+```
+
+#### 3. Authentication ####
+
+If you want to enable authentication to KongMap's UI, it is recommended to run Kongmap behind your Kong Gateway and implement any authentication
+policies you feel is appropriate (OIDC, OAUTH2, Basic Auth, etc.) at the gateway.
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:134]:
```suggestion
policies you feel are appropriate (OIDC, OAUTH2, Basic Auth, etc.) at the gateway.
```

File: app/_hub/yesinteractive/kongmap/index.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+Full documentation available online here: [https://github.com/yesinteractive/kong-map/](https://github.com/yesinteractive/kong-map/)
+
+## Compatibility ## 
+KongMap supports both Kong Open Source and Kong Enterprise Clusters greater than version 1.5.
+
+## Docker Installation ##
+
+Docker image is Alpine 3.11 based running PHP 7.3 on Apache. The container exposes both ports 80 an 443 with a self signed certificated. 
+
+#### 1. Export Cluster Configurations to Variable ####
```
Comment (tlblessing) [inline app/_hub/yesinteractive/kongmap/index.md:84]: Delete the extraneous header markers #.

**Review by tlblessing — CHANGES_REQUESTED** [review]
Suggested edits.

**Comment by nikirago** [comment]
Hi @tlblessing, can you verify the changes are ok? n.

**Review by tlblessing — APPROVED** [review]
https://deploy-preview-2362--kongdocs.netlify.app/hub/yesinteractive/kongmap/

LGTM

**Pull request merged**